core/ustring.cpp

```diff
@@ -142,9 +142,11 @@ void CharString::copy_from(const char *p_cstr) {
 		return;
 	}
 
-	resize(len + 1); // include terminating null char
+	Error err = resize(++len); // include terminating null char
 
-	strcpy(ptrw(), p_cstr);
+	ERR_FAIL_COND_MSG(err, "Failed to copy c-string");
```

Comment:
Should be `err != OK`, and error message should end with a dot. While at it, "C" should be capitalized.

Reply:
Done.